lightkurve/seismology/core.py

```diff
@@ -280,11 +357,167 @@ def plot_echelle(self, deltanu=None, numax=None,
 
             cbar.set_label(ylabel)
             ax.set_xlabel(r'Frequency mod. {:.2f}'.format(deltanu))
-            ax.set_ylabel(r'Frequency [{}]'.format(freq.unit.to_string('latex')))
+            ax.set_ylabel(r'Frequency [{}]'.format(self.periodogram.frequency.unit.to_string('latex')))
             ax.set_title('Echelle diagram for {}'.format(self.periodogram.label))
 
         return ax
 
+    def _make_echelle_elements(self, deltanu, cmap='viridis',
+                         minimum_frequency=None, maximum_frequency=None,
+                         smooth_filter_width=.1,
+                         scale='linear', plot_width=490, plot_height=340, title='Echelle'):
+        """Helper function to make the elements of the echelle diagram for bokeh plotting.
+        """
+
+        if not hasattr(deltanu, 'unit'):
+            deltanu = deltanu * self.periodogram.frequency.unit
+
+        if smooth_filter_width:
+            pgsmooth = self.periodogram.smooth(filter_width=smooth_filter_width)
+            freq = pgsmooth.frequency  # Makes code below more readable below
+            power = pgsmooth.power     # Makes code below more readable below
+        else:
+            freq = self.periodogram.frequency  # Makes code below more readable
+            power = self.periodogram.power     # Makes code below more readable
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'power' is assigned to but never used](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4468966)